schema/schema.sql

```diff
@@ -0,0 +1,73 @@
+-- Creating a Lable Table
+CREATE TABLE "Label" (
+  "id" bigserial,
+  "title" varchar(150),
+  "color" varchar(100),
+  PRIMARY KEY ("id")
+);
+
+-- Creating a Book Table
+CREATE TABLE "Book" (
+  "id" bigserial,
+  "genre_id" int,
+  "author_id" int,
+  "label_id" int,
+  "publish_date" date,
+  "archived" bool,
+  "publisher" varchar(200),
+  "cover_state" varchar(10),
+  PRIMARY KEY ("id"),
+  CONSTRAINT "FK_Book.label_id"
+    FOREIGN KEY ("label_id")
+      REFERENCES "Label"("id")
+);
+
+-- Creating an Authors Table
+CREATE TABLE author (
+    id SERIAL PRIMARY KEY,
+    fist_name VARCHAR(50) NOT NULL,
+    last_name VARCHAR(50) NOT NULL
+);
+
+-- Creating a Game Table
+create TABLE game(
+ id serial primary key,
+ multiplayer boolean not null,
+ last_played date not null,
+ publish_date date not null,
+ archived boolean not null,
+ author_id int references author(id),
+ genre_id int references genre(id),
+ source_id int references source(id),
+ label_id int references label(id)
+);
```

Comment:
- [OPTIONAL] I know it was not mentioned in the project requirement, but it would be nice to create indexes on all foreign keys in this schema based on the requirement. Creating indexes on foreign keys makes queries more efficient. 👍